tsconfig.json

```diff
@@ -4,6 +4,12 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "noImplicitReturns": true,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+    "target": "ES2020",
```

Comment:
Is this going to emit syntax that won't run in current browsers?

Reply:
It won't at the moment; shall I lower it to ensure it does not in the future?